js/field-gmap.js

```diff
@@ -3,15 +3,16 @@
 
 (function($) {
 
-	if ( 'undefined' == typeof google ) {
-		$('.map').css('textAlign','center').append( 'Google Maps API not loaded.' );
-		return;
-	}
-
 	var CMBGmapsInit = function( fieldEl ) {
 
+		var mapCanvas = $('.map', fieldEl ).get(0);
+
+		if ( 'undefined' == typeof google ) {
+			$('<div>'+CMBGmaps.strings.googleMapsApiNotLoaded+'</div>').css({'width':'100%','textAlign':'center','marginTop':'1em'}).appendTo( mapCanvas );
+			return;
+		}
+
 		var searchInput = $('.map-search', fieldEl ).get(0);
-		var mapCanvas   = $('.map', fieldEl ).get(0);
```

Comment:
Please add a space before the opening tick here. And, I know it's not your PR scope, but you could adjust the ones above and below I'd really appreciate it!